templates/Template k8s-patroni.xml

```diff
@@ -48,20 +48,20 @@
                                 <trigger_prototype>
                                     <uuid>8fbd295f2d06451cb11a4f87bab51016</uuid>
                                     <expression>find(/Template k8s-patroni postgres status/k8s-patroni.patroni_archive_command_set[{#CONTEXT},{#NAMESPACE},{#CLUSTER}],1,&quot;regexp&quot;,&quot;true&quot;)=0</expression>
-                                    <name>postgres cluster {#CLUSTER} patroni_archive_command_set NOT true</name>
+                                    <name>Postgres cluster {#CONTEXT}:{#NAMESPACE}:{#CLUSTER} patroni_archive_command_set NOT true</name>
```

Comment:
Should this also be `Patroni postgres cluster member`?

`<description>Discovery of the postgres clusters</description>` <- discovery

Below we have `Patroni postgres member is lagging`

`<description>Discovery of the postgres member statuses</description>` <- discovery